includes/class-wc-gateway-mondido-hw.php

```diff
@@ -239,8 +241,9 @@ public function process_payment( $order_id ) {
 
 			$token_id = isset( $_POST[$token_key] ) ? wc_clean( $_POST['token_key'] ) : 'new';
 
-			delete_post_meta( $order_id, '_mondido_use_store_card');
-			delete_post_meta( $order_id, '_mondido_store_card');
+			$this->orderStorage->delete_meta_data($order, '_mondido_use_store_card');
+			$order->delete_meta_data($order, '_mondido_store_card');
```

Comment:
Why don't we use the order storage here?

Reply:
Mistake, good catch